learning-development/accessibility.qmd

```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
+
 ---
 
-Accessibility tools you can use
+## Assistive technology
 
-If wanting to check anything around colour:
-	- Contrast - webaim / https://jansensan.github.io/color-contrast-matrix/
-	- Colour blindness -
-	- Colour blindness simulation - 
-	- Greyscale - 
+No automated tools fully cover accessibility, manual testing is always required, so if you want to do manual testing have a look at testing out the assistive software commonly used by users yourself.
 
-Helpful browser addins
+Magnifier:
 
-These 'bookmarklets' are bookmarks that instead of saving a URL to a page, save a piece of Javascript code that executes on the page you are looking at, usually to highlight specific mark up. Very nifty!
+- Built into Windows ([guidance for how to turn on Magnifier](https://support.microsoft.com/en-gb/windows/use-magnifier-to-make-things-on-the-screen-easier-to-see-414948ba-8b1c-d3bd-8615-0e5e32204198#:~:text=To%20quickly%20turn%20on%20Magnifier,turn%20on%20the%20Magnifier%20switch.))
+- [ZoomText](https://www.freedomscientific.com/products/software/zoomtext/), available through the [experience lab in DfE](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx)
 
-	- Links
+Screen reader:
 
-Free accessibility tools
-	- Axe devtools, chrome extension / free download, we recommend (and I believe GDS tend to recommend too)
-	- Google Lighthouse, lowest effort, and will catch some basic things, though doesn't cover everything
+- [Non-visual Desktop Access (NVDA) free download](https://www.nvaccess.org/download/)
+- [Job Access With Speech (JAWS)](https://www.freedomscientific.com/products/software/jaws/), available through the [experience lab in DfE](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx)
 
-Worth also noting that no automated tools fully cover accessibility, manual testing is always required, so if you want to do manual testing have a look at:
+Voice activation:
 
-Screen reader
-	- NVDA (free to download)
+- [Dragon](https://www.nuance.com/en-gb/dragon.html), available through the [experience lab in DfE](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx)
 
-Voice activation
-	- Dragon is a paid for licence, though available through the empathy lab
+Along with devices and the software mentioned above, the [experience lab in the Sheffield DfE office](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx) also has a range of other equipment available, including access to a set of vision emulating glasses that you can wear to emulate different visual impairments.
 
-Magnifier
-Built into windows? Check
\ No newline at end of file
+---
```

Comment:
I'd like to see something about cognitive load in here somewhere but doesn't need to be you that adds it. This is a nice and very detailed resource about cognitive load and data visualisations though: https://nightingaledvs.com/cognitive-load-as-a-guide-12-spectrums-to-improve-your-data-visualizations/

Reply:
Made this change in my PR comments commit